Lib/test/test_asyncio/test_base_events.py

```diff
@@ -596,18 +595,11 @@ async def zero_error_coro():
             self.loop.run_forever()
             fut = None # Trigger Future.__del__ or futures._TracebackLogger
             support.gc_collect()
-            if PY34:
-                # Future.__del__ in Python 3.4 logs error with
-                # an actual exception context
-                log.error.assert_called_with(
-                    test_utils.MockPattern('.*exception was never retrieved'),
-                    exc_info=(ZeroDivisionError, MOCK_ANY, MOCK_ANY))
-            else:
-                # futures._TracebackLogger logs only textual traceback
-                log.error.assert_called_with(
-                    test_utils.MockPattern(
-                        '.*exception was never retrieved.*ZeroDiv'),
-                    exc_info=False)
+            # Future.__del__ in Python 3.4 logs error with
+            # an actual exception context
```

Comment:
Can remove reference to Python 3.4 in the comment as well:
```suggestion
            # Future.__del__ in logs error with an actual exception context
```